packages/core/src/Account.ts

```diff
@@ -112,7 +112,7 @@ interface AccountOptions {
   nbPrekeys: number;
 
   /**
-   * Config for MLS and proteus devices. Will fallback to the old cryptobox logic if not provided
+   * Config for coreCrypto in case it supposed to be used. Will fallback to the old cryptobox logic if not provided
```

Comment:
```suggestion
   * Config for coreCrypto in case it's supposed to be used. Will fallback to the old cryptobox logic if not provided
```
the nitpicks are back :sweat_smile:

Reply:
good catch, eagle eye 👁️